awx/ui/src/screens/Instances/Instance.js

```diff
@@ -25,6 +29,27 @@ function Instance({ setBreadcrumb }) {
     { name: t`Details`, link: `${match.url}/details`, id: 0 },
   ];
 
+  const { isK8s, error, isLoading, request } = useRequest(
+    useCallback(async () => {
+      const { data } = await SettingsAPI.readCategory('system');
+      return data.IS_K8S;
+    }, []),
+    false
+  );
+  useEffect(() => {
+    request();
+  }, [request]);
+
+  if (isK8s) {
+    tabsArray.push({ name: t`Peers`, link: `${match.url}/peers`, id: 1 });
+  }
+  if (isLoading) {
+    return <ContentLoading />;
```

Comment:
Does this get used/rendered anywhere?

Reply:
yes as we are waiting on the api request to finish.